cuML/src/knn/knn.cu

```diff
@@ -250,7 +250,7 @@ namespace ML {
 
             float *ptr_d;
             MLCommon::allocate(ptr_d, size_t(length)*size_t(D));
-            MLCommon::updateDevice(ptr_d, ptr+(size_t(chunk_size)*i), size_t(length)*size_t(D));
+            MLCommon::updateDevice(ptr_d, ptr+(size_t(chunk_size)*i), size_t(length)*size_t(D), 0);
```

Comment:
I should probably be creating/accepting a stream here and passing it down to FAISS. I'll create a separate issue to capture this one.